files/usr/share/jeos-firstboot/raspberrywifi

```diff
@@ -56,7 +56,24 @@ is_raspberry()
 # interaction and access to the global systemd_firstboot_args array
 raspberrywifi_systemd_firstboot()
 {
-	if is_raspberry && ls -d /sys/class/net/*/wireless &>/dev/null; then
+        for p in /sys/class/net/* ; do
+                test -f "$p" && continue # skip bonding_masters file
+
+                # Only devices having ID_NET_NAME.* attrs
+                # Ignore errors if udev not available
+                udevadm info -q property -p "$p" 2>/dev/null | grep -qs ID_NET_NAME || continue
+                # But don't touch WLAN interfaces
+                udevadm info -q property -p "$p" | grep -qs "DEVTYPE=wlan" && continue
+
+                d=${p##*/}
+
+                # On Rpi create a configuration file for each interface detected
+                # even when no cable connected
```

Comment:
That needs some discussion:

* Runs even on non-RPi. The `is_raspberry` check is below.
* Why just for the RPi? It's an unexpected difference
* It can be a (negative) surprise in general to have this configured for an unused interface. There should at least be a dialog "DHCP enabled for these interfaces: eth0"

Reply:
Yeah, let's discuss that later then.

Reply:
I changed to have all interfaces configured with DHCP, at least while we don't have a way to re-run the network configuration standalone, and just for RPi as if applied to the virtual ones would be a difference in behaviour (currently we configure just the first one connected).